yarn.lock

```diff
@@ -0,0 +1,15506 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This is a huge autogenerated file.  Do we need this in the repo?

Reply:
Normally, I'd say yes, but for the sample django app, I agree we could omit this. Maybe make a note in the README that usually `yarn.lock` or `package-lock.json` is checked into the repo for the same reason we have `requirements.in` and `requirements.txt` checked in.

Reply:
I was already thinking we should omit requirements.txt as well given you need to regenerate it after resolving feature flags anyways

Reply:
:+1: sounds good to me!

If we do remove it, we should add instructions in the README about needing to generate these and check them into the repo.